### PR TITLE
Improve "Your account" link order in Classic footer

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
This PR will change the default order of the section Your account links in the classic footer.

Default order : Personal info / orders / credit slips / addresses

See Ticket : http://forge.prestashop.com/browse/BOOM-1706

### CODE REVIEW

**Comment by AlexEven** [comment]
Hello @fatmaBouchekoua 
Can you please rebase this PR so that it can be tested and merged?
Thank you!

**Comment by fatmaBouchekoua** [comment]
Hello @AlexEven ,
Done !

**Comment by AlexEven** [comment]
Tested and it's all good, thanks @fatmaBouchekoua !

**Comment by marionf** [comment]
It's also ok for me :)

**Review by eternoendless — APPROVED** [review]

File: ps_customeraccountlinks.php [hunk 1]
```diff
@@ -99,19 +99,19 @@ public function getWidgetVariables($hookName = null, array $configuration = [])
...
             ),
-            2 => array(
+            3 => array(
                 'title' => $this->trans('Credit slips', array(), 'Modules.Customeraccountlinks.Admin'),
                 'url' => $link->getPageLink('order-slip', true),
             ),
-            3 => array(
-                'title' => $this->trans('Addresses', array(), 'Shop.Theme.Global'),
+            4 => array(
+                'title' => $this->trans('Addresses', array(), 'Shop.Global'),
```
Comment (eternoendless) [inline ps_customeraccountlinks.php]: I think you mistakenly changed the domain to an undefined one

**Review by eternoendless — CHANGES_REQUESTED** [review]

**Review by mickaelandrieu — APPROVED** [review]

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @fatmaBouchekoua

**Pull request merged**